### PR TITLE
add the adminhelp command fixes #1804

### LINKED ISSUE #1804: Contact admins
Could we possibly have a thing where whenever you put @ before a message, it contacts all administrators on? /report always isn't reliable, and when someone griefs, but you don't know who it is, you can't use /report. I suggest this be added. It is on various csgo servers and garry's mod servers, and I feel it could work here aswell

### PR DESCRIPTION
**Pull request by notanoperator**
when /report can't always be used fixes https://github.com/TotalFreedom/TotalFreedomMod/issues/1804

### CODE REVIEW

**Comment by notanoperator** [comment]
</img>
Works, and compiles

**Comment by DUFCLiam** [comment]
You forgot the `.java` extension on the `C_adminhelp` class.

**Comment by OxLemonxO** [comment]
Incorrect formatting.

**Comment by Mafrans** [comment]
Why is this needed, we already have /report and noone will be using it anyways.

**Comment by notanoperator** [comment]
@mafrans did you even try reading the issue? smh

File: src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java [hunk 1]
```diff
@@ -123,6 +123,15 @@ public void reportAction(Player reporter, Player reported, String report)
                 playerMsg(player, ChatColor.RED + "[REPORTS] " + ChatColor.GOLD + reporter.getName() + " has reported " + reported.getName() + " for " + report);
             }
         }
+         public void adminHelp(Player sender, String message) {
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java]: Formatting.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@CommandPermissions(level = Rank.OP, source = SourceType.ONLY_IN_GAME, blockHostConsole = true)
+@CommandParameters(description = "Need help from a admin? Use this!", usage = "/<command> <message to admins... >", aliases = "ah,helpme,contactadmins")
+public class Command_adminhelp extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+    
+            if (args.length < 1) {
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java]: Formatting, and why the extra new lines?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    return false;
+}
+    Player player = (Player) sender;
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java]: Just use playerSender..

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        
+        
+        String message = StringUtils.join(ArrayUtils.subarray(args, 0, args.length), " ");
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_adminhelp.java]: Formatting.

**Comment by Wild1145** [comment]
You have deleted all your changes... There are no longer any file changes to this PR.

**Comment by notanoperator** [comment]
@Wild1145 fixed idk why i removed that...

File: src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java [hunk 5]
```diff
@@ -113,7 +113,17 @@ public void adminChat(CommandSender sender, String message)
...
         }
     }
-
+    
+    public void adminHelp(Player sender, String message) {
+       for (Player player : server.getOnlinePlayers())
+        {
+            if (plugin.al.isAdmin(player))
+            {
+                playerMsg(player, ChatColor.RED + "[ADMINHELP] " + ChatColor.GOLD + sender.getName() + ": " + message);
```
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/ChatManager.java:122]: Im concerned that the gold may get confused with the new admin chat colour. Can this be changed to another colour? Dark Blue or a light red maybe?

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please address this and the earlier changes requested.

**Pull request closed without merging**